presentation/src/main/java/com/mashup/alcoholfree/presentation/ui/measuring/MeasuringViewModel.kt

```diff
@@ -1,29 +1,80 @@
 package com.mashup.alcoholfree.presentation.ui.measuring
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.mashup.alcoholfree.domain.usecase.CreateMeasureResultReportUseCase
+import com.mashup.alcoholfree.presentation.ui.home.model.DrinkUiModel
+import com.mashup.alcoholfree.presentation.ui.home.model.MeasureResultReportParamUiModel
+import com.mashup.alcoholfree.presentation.ui.home.model.toDomainModel
 import com.mashup.alcoholfree.presentation.ui.measuring.model.MeasuringState
+import com.mashup.alcoholfree.presentation.utils.Event
 import com.mashup.alcoholfree.presentation.utils.ImmutableList
 import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
 import javax.inject.Inject
 
 @HiltViewModel
 class MeasuringViewModel @Inject constructor(
-
-): ViewModel() {
+    private val createMeasureResultReportUseCase: CreateMeasureResultReportUseCase,
+) : ViewModel() {
     private val _state = MutableStateFlow(initState())
     val state: StateFlow<MeasuringState>
         get() = _state
 
+    private val _createReportSuccessEvent = MutableLiveData<Event<String>>()
+    val createReportSuccessEvent: LiveData<Event<String>>
+        get() = _createReportSuccessEvent
+
+    private val drinkingStartTime = getDateTimeNow()
+    private val drinkingMap = mutableMapOf<String, Int>()
+    private val drinks
+        get() = drinkingMap.toList()
+
+    private fun getDateTimeNow() = LocalDateTime.now().format(DateTimeFormatter.ISO_DATE_TIME)
```

Comment:
Nit: 이거 굳이 함수로 할 이유가 있을까? val로 받아도 괜찮을꺼 같은뎅

Reply:
저거 시작시간이랑 종료시간 두번 써서 함수로 만들긴 했는데 음. 그러게? 이렇게 할까?
```suggestion
    val now: String
        get() =  LocalDateTime.now().format(DateTimeFormatter.ISO_DATE_TIME)
```

Reply:
아 시작,종료 시간을 나눠서 그랬구나. 음 지혜 하고 싶은 대로 하자 ㅎ.. 
코드가 이상한것도 아니고 굳이 고쳐야 될까 싶기도 하넹 담당자 마음대로 하시져~ㅎㅎ
둘다 좋습니다~

Reply:
저거는 코드가 어떤거든 똑같아서 상관없을듯?!